changelog/0050-remove-report-indices.yml

```diff
@@ -0,0 +1,49 @@
+title: Remove report_cases and report_xforms indices
+key: remove-report-indices
+date: 2022-06-17
+optional_per_env: no
+context: |
+  CommCareHQ has two elasticsearch indices "report_cases" and "report_xforms"
+  that were created many years ago for some custom reports, but which haven't
+  been used in some time. This change deletes those indices and the change
+  processors used to populate them. We do not expect this to impact any
+  self-hosted environments, but it's prudent to confirm.
+
+details: |
+  These indices are populated by two change processors ("pillows") -
+  `ReportXFormToElasticsearchPillow` and `ReportCaseToElasticsearchPillow`.
+  These are not referenced in [the sample
+  environment](https://github.com/dimagi/sample-environment), and were
+  officially deprecated in late 2018 in changelog 0007.
+  If enabled, these two pillows do not send data to elasticsearch except for a
+  hardcoded set of project spaces on Dimagi's production environment.
+
+  If your environment does reference these pillows or has index-specific
+  settings configured for those indices, those references will need to be
+  removed.
+
+update_steps: |
+  1. Search your environment's app_processes.yml for `ReportCaseToElasticsearchPillow` and `ReportXFormToElasticsearchPillow`.  If in there, it should look something like this:
```

Comment:
I don't think we need this step. I think the only supported option at this point is the unified `case-pillow`

Reply:
That's what I thought to, but they would still work as far as I can tell, and we never updated the docs appropriately.  We list all the old pillows including these two, and then say:
> For all features to work, each of these ETL processors must appear at least once